src/spawn.ml

```diff
@@ -0,0 +1,46 @@
+module Env = struct
+  type t = string array
+
+  let of_array t = t
+end
+
+external sys_exit : int -> 'a = "caml_sys_exit"
+
+let rec file_descr_not_standard fd =
+  if (Obj.magic (fd : Unix.file_descr) : int) >= 3 then
```

Comment:
Is this supposed to be used under Windows ?

Reply:
Ah, indeed. I updated the code